src/include/daos_types.h

```diff
@@ -155,7 +155,9 @@ typedef struct daos_event {
 		uint64_t	space[20];
 	}			ev_private;
 	/** Used for debugging */
-	uint64_t		ev_debug;
+	uint32_t		ev_debug;
+	/** Abort the task(s) related with the event queue. */
+	uint32_t		ev_abort:1;
```

Comment:
this is a public API change. given also this is an internal change, does not seem to apply to user events, this should not be here.

Reply:
There is a space hole after "ev_error" in original "struct daos_event" for 64-bits alignment.

typedef struct daos_event {
        /** return code of non-blocking operation */
        int                     ev_error;
        /** Internal use - 152 + 8 bytes pad for pthread_mutex_t size difference on __aarch64__ */
        struct {
                uint64_t        space[20];
        }                       ev_private;
        /** Used for debugging */
        uint64_t                ev_debug;
} daos_event_t;

I can add the new field into such hole to avoid the layout changes for "struct daos_event".